assets/questions/mostCommonChar.js

```diff
@@ -0,0 +1,112 @@
+// Copyright 2017 The TIE Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Question data for Most Common Character.
+ */
+
+globalData.questions['mostCommonChar'] = {  // eslint-disable-line dot-notation
```

Comment:
It might not be a bad idea to change this to mostCommonCharacter. We're not being charged by the letter or anything.